lib/bindings/tests/bindings/csharp/Program.cs

```diff
@@ -4,8 +4,8 @@
 try
 {
     var mnemonic = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about";
-
-    var config = BreezSdkLiquidMethods.DefaultConfig(LiquidNetwork.Testnet);
+    var testPubkey = "MIIBazCCAR2gAwIBAgIHPWTUED8FpTAFBgMrZXAwEDEOMAwGA1UEAxMFQnJlZXowHhcNMjQxMDA0MjMxMjM0WhcNMzQxMDAyMjMxMjM0WjAlMQ4wDAYDVQQKEwVCcmVlejETMBEGA1UEAxMKQnJlZXotVGVzdDAqMAUGAytlcAMhANCD9cvfIDwcoiDKKYdT9BunHLS2/OuKzV8NS0SzqV13o4GAMH4wDgYDVR0PAQH/BAQDAgWgMAwGA1UdEwEB/wQCMAAwHQYDVR0OBBYEFNo5o+5ea0sNMlW/75VgGJCv2AcJMB8GA1UdIwQYMBaAFN6q1pJW843ndJIW/Ey2ILJrKJhrMB4GA1UdEQQXMBWBE2h5ZHJhX3lzZUBwcm90b24ubWUwBQYDK2VwA0EAILwfdmryGRMoYX5HotHXci5Z5yTG3ugwdNURG3puXSG2eSXgbmwdzGDJBFTvNsEx9sP0L7nyOaKr3W1Yzf/ECQ==";

```

Comment:
It's a dummy API key I created specifically for tests. Since they're being executed by CI (at least the golang and csharp ones) I don't see another way of going around this but to provide a working key

Reply:
We can inject the API key via secret env var for CI.
Also if the tests are running on testnet we can not require API key for testnet. Both options are ok IMO

Reply:
The second options sounds good, as we're not using Esplora for testnet anyway